helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -105,7 +105,9 @@
 @PreFetchChangedData(enabled = false)
 public class CallbackHandler implements IZkChildListener, IZkDataListener {
   private static Logger logger = LoggerFactory.getLogger(CallbackHandler.class);
+  private static final AtomicLong CALLBACK_HANDLER_UI = new AtomicLong();
```

Comment:
@kaisun2000 This one? `CALLBACK_HANDLER_UI` -> `CALLBACK_HANDLER_UID`

Reply:
good catch. fixed.